https://issues.redhat.com/browse/ACM-4743 Adenf4743 (#5151)
open-cluster-management/rhacm-docs: clusters/release_notes/whats_new.adoc

If you installed {ocp-short} or {product-title-short}, you automatically receive {mce-short}. If any new features exist for {mce-short} install only, you can view them in this section.

> You can now rely on the {olm} `operatorcondition` resource to enforce an efficient {mce-short} upgrade procedure, which will prevent errors from attempts to skip releases. See link:../install_upgrade/upgrade_cluster.adoc#upgrading[Upgrading] for details.

Reviewer: Remember to use [active tone](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense). When I read "rely", i interpreted the procedure to be reliable as if it was a person, which is an example of Anthropomorphism: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

```suggestion
You can now use the {olm} `operatorcondition` resource to enforce the {mce-short} upgrade procedure, which prevent errors from attempts to skip releases. See link:../install_upgrade/upgrade_cluster.adoc#upgrading[Upgrading] for details.
```

Author: Thank you for your review!
